### DISCUSSION TITLE
2 fields in the same line

### DISCUSSION THREAD

**Discussion by lucasbracher** [farridav/django-jazzmin#569] [General]
Good afternoon!
How can I set admin interface so I can show 2 fields in the same line for some given fields in a model?

Thanks!

**Comment by Malith-Rukshan** [comment]
👋 Hello,

To display two fields on the same line in the Django admin interface using the `django-jazzmin` package, you can customize the form layout using the `fieldsets` option. Here’s how you can do it:

1. **Customize the Admin Form Layout:**
   You can override the `AdminForm` for the specific model to display fields on the same line. Here is an example:

   ```python
   from django.contrib import admin
   from .models import YourModel

   class YourModelAdmin(admin.ModelAdmin):
       fieldsets = (
           (None, {
               'fields': (('field1', 'field2'),)  # This displays field1 and field2 on the same line
           }),
           ('Other Information', {
               'fields': ('field3', 'field4'),
           }),
       )

   admin.site.register(YourModel, YourModelAdmin)
   ```

In this example, `field1` and `field2` will be displayed on the same line in the admin form for `YourModel`.

After making these changes, you should see `field1` and `field2` displayed on the same line in the admin interface for `YourModel`.
